connectors/class-connector-acf.php

```diff
@@ -147,13 +149,210 @@ public function action_links( $links, $record ) {
 		return $links;
 	}
 
+	/**
+	 * Tracks the creation of custom field group fields and settings (ACF v5+ only)
+	 *
+	 * @action save_post
+	 *
+	 * @param int     $post_id Post ID.
+	 * @param WP_Post $post    Post object.
+	 * @param bool    $update  Whether this is an existing post being updated or not.
+	 */
+	public function callback_save_post( $post_id, $post, $update ) {
+		// Bail if updating existing post.
+		if ( false !== $update ) {
+			return;
+		}
+
+		// Log new ACF field additions to field groups.
+		if ( 'acf-field' === $post->post_type ) {
+			$parent = get_post( $post->post_parent );
+			if ( $parent ) {
+				$this->log_prop( 'added', $post_id, $post, 'parent', $parent );
+			}
+		} elseif ( 'acf-field-group' === $post->post_type ) {
+			$props = maybe_unserialize( $post->post_content );
+
+			if ( ! empty( $props ) ) {
```

Comment:
@kidunot89 `maybe_unserialize()` does not always return an array. Do you think we need to check that `$props` is an array before running the `foreach` loop?